CHANGES.rst

```diff
@@ -187,6 +187,11 @@ residual_fringe
 
 - Replaced fitting the background with an astropy fitting package [#6739]
 
+saturation
+----------
+
+- New parameter to control how many adjacent-to-saturated pixels are flagged [#6818]
```

Comment:
```suggestion
- New parameter to control how many adjacent-to-saturated pixels are flagged [spacetelescope/stcal#6818]
```

Reply:
I would modify the text to cover the updates to saturation more generally, e.g. "Updated to allow the step to flag neighbors of saturated pixels, which is controlled by the new step param ``n_pix_grow_sat``."  (i.e. we're talking about actual changes to step algorithm, in addition to the simple addition of a step param)